src/policy/fees.h

```diff
@@ -74,6 +74,33 @@ enum FeeEstimateHorizon {
     LONG_HALFLIFE = 2
 };
 
+/* Enumeration of reason for returned fee estimate */
+enum FeeReason {
```

Comment:
shouldn't this be "enum class" to avoid the enum values from leaking into global scope?